gordo_components/server/utils.py

```diff
@@ -301,6 +302,12 @@ def wrapper_method(self, *args, **kwargs):
         # Assign X and y to the request's global context
         g.X, g.y = X, y
 
+        try:
+            logger.debug(f"Size of X: {X.size}, size of y: {y.size}")
+        except Exception:
```

Comment:
If your except-claus assumes that a single exception-type is happening (Y is none), then I think you should only catch that exception which happends when `y` is none.